gcp-test/tests/gcp_kubernetes_node_pool/test-not-found-query.sql

```diff
@@ -0,0 +1,3 @@
+select title, akas, project, location
+from gcp.gcp_kubernetes_node_pool
+where name = '{{ resourceName }}dd'
```

Comment:
The above query is not verifying `GET` response. Please pass all the required parameters to perform the `GET`.